roles/precheck/tasks/main.yml

```diff
@@ -15,6 +15,36 @@
     msg: Please make sure [context.PASSWORD, one_pass, one_token, one_version] are valid strings.
   run_once: true
 
+- name: Validate passwords strength
+  ansible.builtin.assert:
+    that:
+      - item.value | length >= 8
+      - item.value | regex_search('[A-Z]')
```

Comment:
I don't think those constraints are actually correct :thinking: because of https://github.com/OpenNebula/one/blob/master/src/um/User.cc#L364-L384 and https://github.com/OpenNebula/one/blob/master/src/um/User.cc#L27 :thinking: 

For sure one_pass and context.PASSWORD should not be handled in the same way. :point_up::innocent: